src/apps/nem/mosaic/layout.py

```diff
@@ -1,11 +1,16 @@
-from apps.nem.layout import *
-from trezor.messages import NEMTransactionCommon
-from trezor.messages import NEMSupplyChangeType
-from trezor.messages import NEMMosaicDefinition
-from trezor.messages import NEMMosaicCreation
-from trezor.messages import NEMMosaicSupplyChange
-from trezor.messages import NEMMosaicLevy
+from micropython import const
+from trezor import ui
+from trezor.messages import (NEMMosaicCreation, NEMMosaicDefinition,
+                             NEMMosaicLevy, NEMMosaicSupplyChange,
+                             NEMSupplyChangeType, NEMTransactionCommon)
```

Comment:
@jpochyla Not that I'm very keen on this, but leaving one import on one line makes cleaner diffs imho.

Reply:
I'm not either! Lets revisit style later, across the whole codebase.